README.md

```diff
@@ -12,10 +12,11 @@ Application for accounting and business automation in the field of electronic eq
 - JDK11
 - Spring Boot 2
 - Gradle 5
-- Database:
+- Database for development environment:
   - H2 (in memory)
-  - PostgreSQL
-  - MySQL
+- Database for production environment:
+  - PostgreSQL 10
+  - MySQL 5
```

Comment:
Do you really have 2 production databases?

Reply:
Can I have 2 development databases?